src/Components/Dialogs/GetPerson/getPerson.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import style from './getPerson.module.css';
+
+const GetPerson = (props) => {
+  const { items } = props;
+  const listItems = Object.keys(items).map((elem) => <div><NavLink to={`/dialogs/${elem}`} className={style.item}>{items[elem].name}</NavLink></div>);
```

Comment:
Consider separating the code on this line into a number of function or variables.
E.g. `const itemKeys = Object.keys(items);`
The purpose is to improve readability of your code.

Reply:
I removed this function from this  file.